config/config.default.php

```diff
@@ -1,53 +1,66 @@
 
-$mongoUri = getenv('XHGUI_MONGO_URI') ?: '127.0.0.1:27017';
-$mongoUri = str_replace('mongodb://', '', $mongoUri);
-$mongoDb = getenv('XHGUI_MONGO_DB') ?: 'xhprof';
-
 return array(
-    'debug' => false,
-    'mode' => 'development',
+    'debug'     => false,
+    'mode'      => 'development',
 
-    // Can be mongodb, file or upload.
+    // Can be mongodb, file, upload or pdo.
 
     // For file
-    //
-    //'save.handler' => 'file',
-    //'save.handler.filename' => dirname(__DIR__) . '/cache/' . 'xhgui.data.' . microtime(true) . '_' . substr(md5($url), 0, 6),
+    //'save.handler'                    => 'file',
+    //'save.handler.path'               => '/data/profiles/',
+    // provide name to save current run to, or leave empty for default pattern
+    //'save.handler.filename'           => Xhgui_Saver_File::getFilename(__DIR__),
+    //'save.handler.separate_meta'      => true,
+    //'save.handler.meta_serializer'    => 'php',
+
+    // serialize handler for all compatible data: json, igbinary, php. This affects only serialization to files
+    // because mongo handler and db handlers use json for native database support.
+    // Defaults to json. Best performance: 'php'
+    //'save.handler.serializer'        => 'php',
+
+    // for best performance it is recommended to use separate meta file and php as serializer.
 
     // For upload
-    //
     // Saving profile data by upload is only recommended with HTTPS
     // endpoints that have IP whitelists applied.
     //
     // The timeout option is in seconds and defaults to 3 if unspecified.
     //
-    //'save.handler' => 'upload',
-    //'save.handler.upload.uri' => 'https://example.com/run/import',
+    //'save.handler'                => 'upload',
+    //'save.handler.upload.uri'     => 'https://example.com/run/import',
     //'save.handler.upload.timeout' => 3,
 
+
     // For MongoDB
-    'save.handler' => 'mongodb',
-    'db.host' => sprintf('mongodb://%s', $mongoUri),
-    'db.db' => $mongoDb,
-
-    'pdo' => array(
-        'dsn' => 'sqlite:/tmp/xhgui.sqlite3',
-        'user' => null,
-        'pass' => null,
-        'table' => 'results'
-    ),
+    'save.handler'  => 'mongodb',
+    'db.host'       => 'mongodb://127.0.0.1:27017',
+    'db.db'         => getenv('XHGUI_MONGO_DB') ?: 'xhprof',
 
-    // Allows you to pass additional options like replicaSet to MongoClient.
-    // 'username', 'password' and 'db' (where the user is added)
+
+    // for PDO
+    //'save.handler'  => 'pdo',
+    //'db.dsn'        => 'sqlite:/var/www/web/test.sq3',
+
+
+    // authentication for db (for both pdo AND mongo)
+    //'db.user'       => '',
+    //'db.password'   => '',
+
+    // Allows you to pass additional options like replicaSet to MongoClient or pdo settings.
     'db.options' => array(),
-    'templates.path' => dirname(__DIR__) . '/src/templates',
-    'date.format' => 'M jS H:i:s',
-    'detail.count' => 6,
-    'page.limit' => 25,
+    'run.view.filter.names' => array(
```

Comment:
`run.view.filter.names` does not seem to exist. please remove and consider submitting as separate PR.

Reply:
I wanted to keep that file in sync with xhgui config file. I will remove this.